docs/release-notes.md

```diff
@@ -11,6 +11,7 @@ Major changes:
 
 Minor changes:
 
+- providers/openstack{,-metadata}: Add `OPENSTACK_INSTANCE_UUID` attributes
```

Comment:
I think I would opt for this => WDYT? 


- openstack: Add `OPENSTACK_INSTANCE_UUID` attribute
- openstack-metadata: Add `OPENSTACK_INSTANCE_UUID` attribute